src/app/containers/Metadata/index.test.jsx

```diff
@@ -405,6 +431,56 @@ it('should render the twitter metatags', async () => {
   expect(actual).toEqual(expected);
 });
 
+it('should render the default service image as open graph image', async () => {
+  await renderMetadataToDocument();
+  const serviceConfig = services.news.default;
+
+  const ogImage = document
+    .querySelector('head > meta[name="og:image"]')
+    .getAttribute('content');
+  const ogImageAltText = document
+    .querySelector('head > meta[name="og:image:alt"]')
+    .getAttribute('content');
+  const twitterImage = document
+    .querySelector('head > meta[name="og:image"]')
+    .getAttribute('content');
+  const twitterImageAltText = document
+    .querySelector('head > meta[name="og:image:alt"]')
+    .getAttribute('content');
```

Comment:
There is some duplication in here - is there a way of refactoring to make it simpler?